srcopsmetrics/cli.py

```diff
@@ -46,7 +45,13 @@ def get_entities_as_list(entities_raw: Optional[str]) -> List[str]:
 
 @click.command()
 @click.option(
-    "--repository", "-r", type=str, required=False, help="Repository to be analysed (e.g thoth-station/performance)",
+    "--repository",
+    "-r",
+    type=str,
+    required=False,
+    help="""Repository to be analysed (e.g thoth-station/performance)
```

Comment:
If they enter spaces after comma accidentally, what happens?

Reply:
it would be accepted as different arguments, and therefore considered invalid options by `click`

Reply:
where is considered invalid by click? `'thoth-station/common, thoth-station/repo'`

Reply:
aaaah  you meant like this! This is considered invalid format, because it will be passed as one string and it won't be processed. Currently you don't have to enclose the multiple repos as one string, so instead of doing `-r  'foo, bar'` you will do `-r foo,bar`.

The same format is supported in entities, where you can do `-e Issue,PullRequest` and it'll be same as `-e Issue -e PullRequest`

Reply:
It is considered invalid format receiving an error?

Reply:
> It is considered invalid format receiving an error?

https://github.com/thoth-station/mi/pull/361/commits/1476f6d1f9f18c8d6ffb9d2a2382d29b27b6246a
trailing whitespaces are removed now, so it is possible to do `-r 'foo, bar'` as well as other whitespace combinations, as long as the repositories names are correct and are separated by comma